Portfolio/index.html

```diff
@@ -6,7 +6,67 @@
     <link rel="stylesheet" href="index.css" />
   </head>
 
-  <body style="margin: 0; padding: 0">
+  <body style="margin: 0; padding: 0; font-family: 'Arial'">
+    <h1>Khan</h1>
+    <ul>
+      <li>About</li>
+      <li>Skills</li>
+      <li>Project</li>
+      <li>Reviews</li>
+    </ul>
+    <span><img src="/phone.icon.png" alt="call-icon" /></span><br />
+    <button>CONTACT ME</button>
+    <h1>Nihal Ahmad</h1>
+    <span><img src="/johnDoe.png" alt="my-image" /></span>
+    <h1>Front End Developer</h1>
+    <h2>Get to Know Me</h2>
+    <p>
+      I am a seasoned front end web developer with a passion for crafting
+      exceptional digital experiences. With over 2 years of hands-on
+      experience,i bring the deep proficiency in HTML, CSS,Javascript,TypeScript
+      and a keen mastery of the modern framework like React.js node.js etc. I
+      specialised in making responsive , interactive and user appealing user
+      interfaces. Additionally, my proficiencies in firebase ensure secure and
+      efficient backend integration, makimg me a versatile and dynamic developer
+      ready to tackle any project.
+    </p>
+    <br />
+    <p>
+      Name : Nihal Ahmad <br />
+      Age : 20 <br />
+      Experience : 2 years
+    </p>
+    <h2>Skills</h2>
+    <span><img src="/html.icon.png" alt="html" /><br />HTML<br />
+    <span><img src="/css.icon.png" alt="css" /></span><br />CSS <br>
+    <span><img src="/js.icon.png" alt="js" /></span><br />JavaScript<br>
+    <span><img src="/ts.icon.png" alt="ts" /></span><br />TypeScript<br>
+    <span><img src="/svelte.icon.png" alt="svelte" /></span><br />Svelte.js<br>
+    <span><img src="/firebase.icon.png" alt="firebase" /></span><br />Firebase<br>
+    <span><img src="/react.icon.png" alt="react" /></span><br />React.js <br> React native <br>
+    <h2>GET IN TOUCH</h2>
+    <H3>CONTACT</H3>
+    <form action="mailto:nihalahmad19453@gmail.com" method="GET">
+      Name: <input type="text" name="name" placeholder="Enter Name"><br>
+      Email: <input type="email" name="email" placeholder="Enter E-mail"><br>
```

Comment:
Please add some space or line breaks between the form input fields for better readability.